mlrun/api/crud/model_monitoring/helpers.py

```diff
@@ -127,3 +127,27 @@ def get_stream_path(project: str = None):
     return mlrun.common.model_monitoring.helpers.parse_monitoring_stream_path(
         stream_uri=stream_uri, project=project
     )
+
+
+def get_connection_string(project: str = None) -> str:
```

Comment:
Why is the default `None` for `project`? It won't work without a project.
Also, it doesn't return a `str` anymore - need to change the hint.